cmd/cloud/server.go

```diff
@@ -64,6 +64,8 @@ func init() {
 	serverCmd.PersistentFlags().Bool("keep-database-data", true, "Whether to preserve database data after installation deletion or not.")
 	serverCmd.PersistentFlags().Bool("keep-filestore-data", true, "Whether to preserve filestore data after installation deletion or not.")
 	serverCmd.PersistentFlags().Bool("require-annotated-installations", false, "Require new installations to have at least one annotation.")
+
+	serverCmd.PersistentFlags().String("utility-group-values", "", "Gitlab repository from which to fetch utility values")
```

Comment:
I'm testing with https://gitlab.com/gigawhitlocks/utility-group-values provided on the command line, but I need someone to create this repository under the gitlab.com/mattermost username, or the github.com/mattermost username. The latter would require a small change to `buildValuesPath` since right now it assumes Gitlab's URL format but if we prefer to host these values on GitHub, I'm 0/5 out of where they live, they just need to be somewhere the Provisioner will have network access at runtime.